gluster/cli/peer.py

```diff
@@ -47,6 +47,32 @@ def detach(host):
     cmd = ["detach", host]
     return peer_execute(cmd)
 
+def detach_all():
+    """
+    Removes All Hosts from Cluster
+
+    :returns: nothing
```

Comment:
:returns: Output of peer detach commands for all peers, raises
GlusterCmdException((rc, out, err)) on error